app/src/main/java/org/mozilla/tv/firefox/navigationoverlay/NavigationOverlayFragment.kt

```diff
@@ -134,13 +133,11 @@ class NavigationOverlayFragment : Fragment() {
     private lateinit var serviceLocator: ServiceLocator
     private lateinit var navigationOverlayViewModel: NavigationOverlayViewModel
     private lateinit var toolbarViewModel: ToolbarViewModel
-    private lateinit var pocketViewModel: PocketViewModel
     private lateinit var hintViewModel: HintViewModel
     private lateinit var toolbarUiController: ToolbarUiController
 
     private var channelReferenceContainer: ChannelReferenceContainer? = null // references a Context, must be nulled.
     private val pinnedTileChannel: DefaultChannel get() = channelReferenceContainer!!.pinnedTileChannel
-    private val pocketChannel: DefaultChannel get() = channelReferenceContainer!!.pocketChannel
```

Comment:
Can you file a follow-up to remove the remaining pocket assets (so we can reduce the APK size) and references in the code?

Reply:
Filed https://github.com/mozilla-mobile/firefox-tv/issues/2998